superset-frontend/src/dashboard/styles.ts

```diff
@@ -18,6 +18,39 @@
  */
 import { css, SupersetTheme } from '@superset-ui/core';
 
+export const headerStyles = (theme: SupersetTheme) => css`
```

Comment:
Hmm... I wonder if these are worth moving into (or along side of) the theme itself for global re-use

Reply:
Previously it targeted only Dashboard, so I moved it here in order to reduce the impact and chances of regressions. You're right though that we might want to make it global